src/hotspot/share/memory/metaspace/metachunk.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Not RedHat?

Reply:
I only add my company when I consider the change substantial enough.